package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "gatsby clean",
     "format": "prettier --write \"**/*.{js,jsx,json,md,ts,tsx}\"",
     "type-check": "tsc --noEmit",
-    "lint": "tslint --project .",
+    "lint": "eslint --fix './{src,stories}/**/*.{ts,tsx,js,jsx}'",
```

Comment:
Osobně bych asi nedával `--fix`, ale přidal samotný script `lint:fix`, který budu moc pustit. Myslím si, že je lepší, když se člověk učí psát kód rovnou správně formátovaný, než když ho může napsat jakkoli a on se magicky naformátuje někde commit hookem.

Ale je to dost moje osobní preference :-)

Reply:
Díky, naprostý souhlas, opraveno.